### PR TITLE
hopefully final fix for location name lookup

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?186192

@calellowitz / up for grabs

### CODE REVIEW

**Comment by czue** [comment]
(looking at tests)

File: corehq/apps/importer/tests.py [hunk 1]
```diff
@@ -334,3 +335,27 @@ def testLocationOwner(self):
         error_message = ImportErrors.InvalidOwnerId
         self.assertIn(error_message, res['errors'])
         self.assertEqual(res['errors'][error_message]['rows'], [5])
+
+
+class ImporterUtilsTest(SimpleTestCase):
```
Comment (czue) [inline corehq/apps/importer/tests.py]: i didn't bother adding an explicit test for location owners because it's already tested by the test that failed

**Comment by czue** [comment]
@esoergel might be interested in importer tweaks - thanks for the tests that caught this!

**Comment by czue** [comment]
(would be nice to get this merged before eod if possible and no review issues)

**Comment by esoergel** [comment]
Looks good to me

**Pull request merged**